plone/app/z3cform/converters.py

```diff
@@ -304,9 +304,9 @@ def toFieldValue(self, value):
         return collectionType(untokenized_value)
 
 
-@adapter(IRelation, IRelatedItemsWidget)
-class RelationChoiceRelatedItemsWidgetConverter(BaseDataConverter):
```

Comment:
shouldn't we additionally keep the `IRelatedItemsWidget` adapted converter for BBB, for those using the related items widget instead the new content browser?